client/styles/fraTable.scss

```diff
@@ -142,6 +142,10 @@
   right: 0;
 }
 
+.fra-table__review-cell {
+  width: 32px;
```

Comment:
use spacing-m

Reply:
the problem is that fra-table__review-cell class is under styles/fraTable.css and spacing-m is under style/_partials/spacing.

another strange thing is that config.scss is replicated, like buttons.scss and status.scss...

<img width="207" alt="Schermata 2022-04-28 alle 09 18 25" src="https://user-images.githubusercontent.com/10047945/165698426-468e63a4-7393-4dfc-a069-51bcfe47aca7.png">